### DISCUSSION TITLE
ESP32-S3 USB audio?

### DISCUSSION THREAD

**Discussion by LPJensen** [hathach/tinyusb#1388] [General]
I'm doing a project where we need an ESP32-S3 to act as a USB audio interface, preferably stereo in/stereo out. But for now we would be happy just to get stereo in up and running.
We got the UAC2 Headset example to build and run, but it seems that handling of interrupt events from the host is failing every other interrupt, and we only get audio data in every second frame.
I know this might not be a TinyUSB related problem, but has anybody succeeded in getting USB audio running correctly on ESP32-S3 and would like to share some insights?

Thanks!

**Comment by hathach** [comment]
Similar issue is discussed here https://github.com/hathach/tinyusb/issues/1382
